iquip/apps/scheduler.py

```diff
@@ -223,6 +258,39 @@ def sizeHint(self, option: QStyleOptionViewItem, index: QModelIndex) -> QSize:
         return experimentView.sizeHint()
 
 
+class SchedulerPostWorker(QObject):
+    """Worker for posting a request to the proxy server, targeting the scheduler.
+
+    Signals:
+        done: The signal is emitted when the procedure of the worker is done.
+
+    Attributes:
+        mode: The type of command that is requested to the server.
+        rid: The run identifier value of the target experiment.
+    """
+    done = pyqtSignal()
+
+    def __init__(self, mode: str, rid: int):
+        """Extended.
+
+        Args:
+            mode: The type of command that is requested to the server.
+            rid: The run identifier value of the target experiment.
+        """
+        super().__init__()
+        self.mode = mode
+        self.rid = rid
+
+    def run(self):
+        """Overridden."""
+        base_path = "http://127.0.0.1:8000/experiment/"
```

Comment:
Well, it is quite confusing to use `PyQt` since they use camel case, but we decided to use camel case near the Qt-context.

See also: snu-quiqcl/qiwis#2